src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.elastic.search.recipe.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "Blog.Elasticsearch.Search",
+  "displayName": "Blog - Elasticsearch Search",
+  "description": "Creates Elasticsearch settings, permission and indices.",
+  "author": "The Orchard Core Team",
+  "website": "https://orchardcore.net",
+  "version": "1.0.0",
+  "issetuprecipe": false,
+  "categories": [ "default" ],
+  "tags": [ "blog", "Elasticsearch" ],
+
+  "steps": [
+    {
+      "name": "feature",
+      "enable": [
+        "OrchardCore.Search.Elasticsearch"
```

Comment:
Also enable Search (and in the Lucene recipe too)?

Reply:
done

Reply:
Failing to run Unit Tests when adding it. Will try to find why. Really strange one.